### PR TITLE
switched to clientx from layerx

### PR DESCRIPTION
**Pull request by cchan987**
addresses #470

### CODE REVIEW

File: nengo_gui/static/netgraph.js [hunk 1]
```diff
@@ -150,8 +150,10 @@ Nengo.NetGraph = function(parent, args) {
 
             self.menu.hide_any();
 
-            var x = (event.layerX / $(self.svg).width())
-            var y = (event.layerY / $(self.svg).height());
+            var x = (event.clientX / $(self.svg).width())
+            var y = (event.clientY / $(self.svg).height());
+
+            console.log(event)
```
Comment (jgosmann) [inline nengo_gui/static/netgraph.js]: debug code to remove

File: nengo_gui/static/netgraph.js [hunk 2]
```diff
@@ -179,7 +181,7 @@ Nengo.NetGraph = function(parent, args) {
                 scale = 1. / scale;
             }
 
-            Nengo.Component.save_components();
+            //Nengo.Component.save_components();
```
Comment (jgosmann) [inline nengo_gui/static/netgraph.js]: Why is this commented out?

**Comment by jgosmann** [comment]
This works now much better. I noticed that the zooming anywhere (not just over plots) is not centered on the mouse pointer along the vertical axis. Probably the height of the toolbar isn't respected in the calculations. Though, you might prefer to open a separate issue for that.

Another note: Apparently we have to places where the zooming is implemented (netgraph items and plots)? It might be worthwhile to do some refactoring to remove that code duplication. But that is probably another PR, too.

**Comment by cchan987** [comment]
updated

**Comment by tcstewar** [comment]
Hmm, this gives very wrong zooming for me (Chrome on Windows).  

For example, if I just open up default.py and put my mouse cursor inside on of the circles in the outer ring of the Ensemble and try scrolling, my mouse cursor fairly quickly is outside the circle.  So that means it's not zooming based on where the mouse cursor is located.  A similar effect seems to be happening for Value graphs and other Components.

**Comment by cchan987** [comment]
> this gives very wrong zooming for me (Chrome on Windows).
This looks to be an issue on master as well actually.

**Comment by tcstewar** [comment]
> This looks to be an issue on master as well actually. Perhaps it should be a seperate PR?

Chrome on Windows in master is working fine for me....

**Comment by jgosmann** [comment]
> This looks to be an issue on master as well actually. Perhaps it should be a seperate PR?
> 
> Chrome on Windows in master is working fine for me....

Firefox on OS X in master is working fine for me too.

**Comment by cchan987** [comment]
Ah, It seems to be a discrepancy in the mouse Y location caused by the toolbar. hunting for fix...

**Comment by cchan987** [comment]
Updated, this seems to be working for me now.

File: nengo_gui/static/top_toolbar.js [hunk 3]
```diff
@@ -53,6 +53,8 @@ Nengo.Toolbar = function(filename) {
 
     this.toolbar = $('#toolbar_object')[0];
 
+    $(this.toolbar).css('height', '2.5em')
```
Comment (jgosmann) [inline nengo_gui/static/top_toolbar.js]: This should not be necessary, I think.

**Comment by tcstewar** [comment]
This is now looking good to me.  I'm going to squash the commits together before merging though (mostly as practice because I've never done that before).

**Pull request merged**